docs/application/tizen-studio/rt-ide/getting-started/overview.md

```diff
@@ -55,21 +56,20 @@ Check the following prerequisites before installing the Tizen Studio for RT:
 </tr>
 </table>
 
-
-- RT platform requirements
+- RT platform requirements:
```

Comment:
Is the ":" really needed here? There is an additional sentence for this list item, which is not a sample code or a list.
For now from here every sentences are finished with : except the last one. 

Same question to line 66 and 72.